app/components/dato/link.rb

```diff
@@ -8,4 +8,23 @@ def initialize(node, root)
   def generated_tag
     "a"
   end
+
+  def link_attributes
+    attr = {
+      "href" => @node.url,
+      "class" => "dato-cms-#{@node.type}",
+    }
+    %w[rel target].each do |type|
+      value = extract_meta(type)
+      attr[type] = value if value
```

Comment:
is it necessary to check `if value`? could we simply always set it? maybe compact empty values at the end?

Reply:
yes true! when it's nil, its sent as nil :D